versions/next/asyncapi.md

```diff
@@ -17,24 +17,25 @@ The AsyncAPI Specification is a project used to describe and document Asynchrono
 The AsyncAPI Specification defines a set of files required to describe such an API.
 These files can then be used to create utilities, such as documentation, integration and/or testing tools.
 
-The file(s) MUST describe the operations a new [process](#definitionsProcess) can perform. For instance:
+The file(s) MUST describe the operations a new [application](#definitionsApplication) can perform. For instance:
```

Comment:
What do you think about changing the point of view here?

For instance:
```suggestion
The file(s) MUST describe the operations an [application](#definitionsApplication) performs. For instance:
```

So, instead of looking at it from the consumer/client perspective, we use the AsyncAPI document to define what the application actually does.

Reply:
Agreed, will fix